components/radix-platform/radix-cluster-cleanup/helmRelease.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: radix-cluster-cleanup
+  namespace: default
```

```suggestion
  namespace: radix-cluster-cleanup
```
Install radix-cluster-cleanup in its own namespace